code/datums/components/bump_attack.dm

```diff
@@ -89,9 +89,10 @@
 	var/mob/living/living_target = target
 	if(bumper.faction == living_target.faction)
 		return //FF
-	var/mob/living/carbon/human/human_target = target
-	if(bumper.wear_id.iff_signal == human_target.wear_id.iff_signal)
-		return //FF
+	if(ishuman(target) && (bumper.wear_id != null))
+		var/mob/living/carbon/human/human_target = target
+		if((human_target.wear_id != null) && (bumper.wear_id.iff_signal == human_target.wear_id.iff_signal))
```

Comment:
no, you need to just use ?.

Reply:
You probably still want to check if the bumper has ID, since it will otherwise count as IFF between 2 randoms without ID, since they both match as null.